img/flutter.svg

```diff
@@ -0,0 +1,19 @@
+<svg width="27" height="32" viewBox="0 0 27 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
just wondering, are we not minifying svg files in this project?
cc: @adam-deriv

Reply:
good question. I've checked and vite build through esbuild minifies those assets for us on production